### DISCUSSION TITLE
CEFSharp and windows SSO

### DISCUSSION THREAD

**Discussion by mbowles201** [cefsharp/CefSharp#4591] [General]
We are using a much older version of CEFSharp to pop a window out of a wnforms app to display an app that should recognize you are an auth'd user of DevOps and smoothly let you in.   This works using the WebView2 control.   We want to stick with CEFSharp and upgrade to the latest, but this is a sticking point.
I found another discussion that provides this link saying "if you are using the Chromium runtime, you should be ok or just need to add the Windows User extension"  https://bitbucket.org/chromiumembedded/cef/issues/2969/support-chrome-windows-with-cef-callbacks
but I can't get access to it.  
Has anyone gotten this to work?

**Comment by amaitland** [comment]
See https://github.com/cefsharp/CefSharp/discussions/4552#discussioncomment-6569095

Try the `Chrome Runtime` with the latest stable version (`M116`) and see how you go. If it doesn't work then you'll probably need to follow up on https://www.magpcss.org/ceforum/index.php

**Reply by mbowles201** [reply]
Apologies, I meant to say "can't get access to it".  I will seek out the runtime and see what I can do with it.

**Reply by mbowles201** [reply]
The BitBucket link here, I am also restricted from.
![image](https://github.com/cefsharp/CefSharp/assets/3237002/c43b9ce2-2529-432c-afca-fe3dbde6ce54)
Do you have a source with instructions for use the Chrome Runtime with CEFSharp?

**Reply by mbowles201** [reply]
I implemented CefSettings with ChromeRuntime = true in my Program.cs file.   When I run my app (which waits until a user clicks a certain button to launch the CefBrowser with selected URL) it kills/closes the form after 12 seconds on its own.  If I set the settting prior to new ChromiumWebBrowser, I get 5-6 instances of my main form suddenly and they keep growing until I kill the app.
I am doing a simple 
```
var browser = new ChromiumWebBrowswer(url);
this.Controls.Add(browser);
```

nothing more.

**Reply by mbowles201** [reply]
I found this: https://github.com/cefsharp/CefSharp/discussions/3469
But it ends directing to #4123 and my bitbucket account comes up with no access to that bitbucket project.

**Reply by mbowles201** [reply]
Got an example that sets the ChromeRuntime to true and doesnt crash, but my SSO still doesnt run.  May try requesthandler and kerberos token.

**Comment by amaitland** [comment]
> But it ends directing to #4123 and my bitbucket account comes up with no access to that bitbucket project.

The CEF issue tracker moved to `GitHub`, I've updated the remaining links in #4123

https://www.magpcss.org/ceforum/index.php would be the best place to ask to guidance or what is/isn't supported.
